### ISSUE TITLE
Sobre o tempo entre a marcação dos jogos.

### ISSUE THREAD

**Issue by pedromuniz378** [guynovaes/INJOLOCA#2]
Olá Guy, hoje tive um problema na hora de enviar meus jogos para o site da loteria caixa. O problema consistia em o INJOLOCA não esperar o site carregar um volante, e em seguida já carregar outro e outro. Na prática, eu mandava carregar 50 jogos, mas apenas cerca de 23 eram de fato carregados.
Com isso, tomei a liberdade de mudar o seu código e aumentei o tempo de espera da marcação de um jogo para outro para 750 ms, 500 ms já bastavam, mas preferi adicionar uma margem para não ocorrer o mesmo problema de novo. Minha mudança foi na linha 246 do arquivo popup.js, onde você chama a função sleepFor. Se julgar útil meu comentário, favor adicionar em futuras versões.

Por fim, quero te parabenizar pelo ótimo trabalho com os softwares COLOGA e INJOLOCA, você fez e está fazendo um trabalho incrivel.

**Comment by guynovaes** [comment]
@pedromuniz378  sim, percebi isso também. Colocando o número acima de 500ms fica bom para dar tempo o site ler todos os jogos.
